Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -302,6 +302,25 @@ def condition_(self, fragment, stepList, key, hasHarvest):
     '--procModifiers': 'trackingMkFit'
 }
 
+#DeepCore seeding for JetCore iteration workflow
+class UpgradeWorkflow_seedingDeepCore(UpgradeWorkflowTracking):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
+    def condition_(self, fragment, stepList, key, hasHarvest):
+        return '2021' in key or '2024' in key
```

Comment:
this limits the setup to 2021 and 2024 workflows and explains the reason of missing 11723.17

Reply:
oh, nevermind, is 11723 meant to be 2021?

Reply:
Why? 11723.0 is a 2021 workflow, isn't so?

Reply:
it looks like I confused myself with "2017" in relval_2017.py , which is just a number in this context and not an actual 2017

Reply:
yes, it happened also to me yesterday! not the best file name

Reply:
You based this special workflow on `UpgradeWorkflowTracking`, which only adds special workflows for ttbar: https://github.com/cms-sw/cmssw/blob/b91d4a5dba8db0c252d8093a7c7c9cc84716138f/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L183

Reply:
thank you @kpedro88, but so why if I redefine my workflow like that I continue to not see any QCD-DeepCore workflow? What Is wrong in these lines?

```
class UpgradeWorkflow_seedingDeepCore(UpgradeWorkflow):
    def condition(self, fragment, stepList, key, hasHarvest):
        result = (fragment=="QCD_Pt_1800_2400_14TeV") and ('2021' in key or '2024' in key) and hasHarvest and self.condition_(fragment, stepList, key, hasHarvest)
        if result:
            # skip ALCA and Nano
            skipList = [s for s in stepList if (("ALCA" in s) or ("Nano" in s))]
            for skip in skipList:
                stepList.remove(skip)
        return result
    def condition_(self, fragment, stepList, key, hasHarvest):
        return True
upgradeWFs['seedingDeepCore'] = UpgradeWorkflow_seedingDeepCore(
    steps = [
        'Reco',
        'RecoGlobal',
    ],
    PU = [],
    suffix = '_seedingDeepCore',
    offset = 0.17,
)
upgradeWFs['seedingDeepCore'].step3 = {
    '--procModifiers': 'seedingDeepCore'
}
```

Reply:
You have to use the "short name" of the fragment, third argument here: https://github.com/cms-sw/cmssw/blob/b91d4a5dba8db0c252d8093a7c7c9cc84716138f/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1214

Reply:
oh, thanks! now everything went fine! :)